ch_pipeline/core/containers.py

```diff
@@ -802,6 +802,39 @@ def source(self):
         return self.index_map["source"]
 
 
+class GaltAutocorrelation(ContainerBase):
+    _axes = ("freq", "pol", "time")
```

Comment:
To support common axes like `freq` and `time`, there are base classes in `draco` that you should inherit from. `draco.core.containers.FreqContainer` and `draco.core.containers.TODContainer`.